demo/cca/manual-end-to-end.md

```diff
@@ -0,0 +1,295 @@
+# CCA Demonstration Steps
+
+This document explains all the steps one need to follow to successfully run
+end to end CCA Platform Verification Demonstration.
+
+## Preconditions
+
+### Install on Ubuntu 
+* One needs to install `Go`, `jq`,`sqlite3`,`tmux` and `curl`.
+
+Use the below mentioned path to install go in your system:
+
+`https://go.dev/doc/install`
+
+Installing jq:
+```sh
+sudo apt update && sudo apt install jq && jq --version
+```
+
+Installing sqlite3:
+```sh
+sudo apt update
+sudo apt install sqlite3 
+sqlite3 --version
+```
+You will get an output like this:
+`3.31.1 2020-01-27 19:55:54 3bfa9cc97da10598521b342961df8f5f68c7388fa117345eeb516eaa837balt1`
+
+Installing curl:
+```sh
+sudo apt install curl
+```
+
+Installing tmux:
+```sh
+sudo apt-get install tmux
+```
+
+
+* For build to succeed one needs to install following packages:
+1. protoc-gen-go with version v1.26
+From  `https://github.com/protocolbuffers/protobuf/releases` download the `protobuf-all-[VERSION].tar.gz`.
+Extract the contents and change in the directory
+
+```sh
+./configure
+make
+make check
+sudo make install
+```
+To check if this works
+`protoc --version`
+
+2. protoc-gen-go-grpc version v1.1
+`go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1`
+3. protoc-gen-go-json version v1.1.0
+`go install github.com/mitchellh/protoc-gen-go-json@v1.1.0`
+4. mockgen version v1.6.0
+`go install github.com/golang/mock/mockgen@v1.6.0`
+
+### Other systems
+
+* One need to install jq and curl.
+
+
+* For build to succeed one needs to install following packages:
+1. protoc-gen-go with version v1.26
+2. protoc-gen-go-grpc version v1.1
+3. protoc-gen-go-json version v1.1.0
+4. mockgen version v1.6.0
+
+* Commands below assume execution in a Bourne-compatible shell. Please adjust appropriately in case any other shell is used.
+
+
+## Creation of CCA Endorsements
+
+First create new Concise Module Identifiers (CoMID's) and use them in creating Concise Reference Integrity Manifests (CoRIM's) using reference templates located under `docs/demo/cca/prov-verif-e2e/data/templates` to provision them in Veraison Verification Service.
+
+More details about CoRIM and CoMID can be found [here](https://datatracker.ietf.org/doc/draft-ietf-rats-corim/)
+
+### Initial Setup
+
+Do this ONLY if it is not done already:
+PLEASE NOTE: Setup instructions are same between PSA and CCA demonstration.
+
+In a new bourne shell session
```

Comment:
```suggestion
In a new Bourne-compatible shell session
```
(_extremely_ nit-picky note: `bash` is the Bourne Again SHell. Bourne shells don't really exist on modern systems, though `bash` can be forced to emulate a  Bourne shell by invoking it via `/bin/sh` symlink; however, this should not be necessary here.)